README.md

```diff
@@ -1 +1,132 @@
-# golang-ic
+[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
+
+## Introduction
+
+> *App introduction goes here ...*
+
+## Project Setup
+
+### Prerequisites
+
+- [Go - 1.20](https://golang.org/doc/go1.20) or newer
+
+
+
```

Comment:
This might be the problem from the templates, but it's better to remove these extra lines